types/backbone/index.d.ts

```diff
@@ -515,7 +518,7 @@ declare namespace Backbone {
         decodeFragment(fragment: string): string;
         getSearch(): string;
         stop(): void;
-        route(route: string | RegExp, callback: Function): number;
+        route(route: string | RegExp, callback: (fragment: string) => void): number;
```

Comment:
```suggestion
        route(route: string | RegExp, callback: RouterCallback): number;
```

Reply:
`History` callback differs from `Router` callback. 

`route` is here: 
https://github.com/jashkenas/backbone/blob/master/backbone.js#L1933

Usage:
https://github.com/jashkenas/backbone/blob/master/backbone.js#L1961

Reply:
You are absolutely right, I stand corrected. This line also proves your point.

https://github.com/jashkenas/backbone/blob/153dc41616a1f2663e4a86b705fefd412ecb4a7a/backbone.js#L1676